src/app/map/[id]/Tile/hierarchy-tile.static.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import { useRouter, useSearchParams } from "next/navigation";
+import type { HexTileData } from "../State/types";
+import { StaticBaseTileLayout } from "./base.static";
+import { startTransition } from "react";
+import { getColorFromItem } from "./item.static";
+
+interface HierarchyTileProps {
+  item: HexTileData;
+  isLast: boolean;
+  hierarchy: HexTileData[];
+  itemIndex: number;
+}
+
+const _createNavigationUrl = (
+  item: HexTileData,
+  currentSearchParams: URLSearchParams,
+  hierarchy: HexTileData[],
+  itemIndex: number,
+): string => {
+  const newParams = new URLSearchParams(currentSearchParams);
+
+  // Get all hierarchy items up to and including the clicked item
+  const hierarchyUpToItem = hierarchy.slice(0, itemIndex + 1);
+
+  // Extract their database IDs for the expandedItems
+  const hierarchyIds = hierarchyUpToItem.map(
+    (hierarchyItem) => hierarchyItem.metadata.dbId,
+  );
+
+  // Get any existing expanded items that are not part of the hierarchy
+  const existingExpandedItems = newParams.get("expandedItems");
+  const existingIds = existingExpandedItems
+    ? existingExpandedItems.split(",")
+    : [];
+
+  // Combine hierarchy IDs with existing expanded items (remove duplicates)
+  const allExpandedIds = [...new Set([...hierarchyIds, ...existingIds])];
+
+  // Create the new URL with expanded hierarchy items
+  const baseUrl = `/map/${item.metadata.dbId}`;
+
+  if (allExpandedIds.length > 0) {
+    return `${baseUrl}?expandedItems=${allExpandedIds.join(",")}`;
+  }
+
+  return baseUrl;
+};
+
+export const HierarchyTile = ({
+  item,
+  hierarchy,
+  itemIndex,
+}: HierarchyTileProps) => {
+  const router = useRouter();
+  const searchParams = useSearchParams();
+
+  const scale = 1; // Fixed scale for hierarchy tiles
+  const baseHexSize = 50; // Increased size for better visibility
```

Comment:
Define constants HIERARCHY_TILE_BASE_HEXSIZE = 60 and HIERARCHY_TILE_SCALE = 1 in src/app/map/page/[id]/constants.ts and use them here.